kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -47,6 +47,7 @@ function _contentState(data) {
   } else {
     progress = data.progress_fraction;
   }
+  const assessmentMetaData = data.asessmentmetadata || {};
```

Comment:
* when [] is used, it will assign `undefined` to assessmentMetaData, and I think that's actually desirable here.
* no need for conditionally-available randomize, model `AssessmentMetaData` gives it a default value `randomize = models.BooleanField(default=False)`